src/main/java/mate/jdbc/dao/DriverDaoImpl.java

```diff
@@ -0,0 +1,129 @@
+package mate.jdbc.dao;
+
+import java.sql.Connection;
+import java.sql.PreparedStatement;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.sql.Statement;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Optional;
+import mate.jdbc.exception.DataProcessingException;
+import mate.jdbc.lib.Dao;
+import mate.jdbc.model.Driver;
+import mate.jdbc.util.ConnectionUtil;
+
+@Dao
+public class DriverDaoImpl implements DriverDao {
+    private static final String createDriverQuery
+            = "INSERT INTO drivers(name, license) VALUES(?, ?)";
+    private static final String getDriverQuery
+            = "SELECT * FROM drivers WHERE id = ? AND is_deleted = false";
+    private static final String getAllDriversQuery
+            = "SELECT * FROM drivers WHERE is_deleted = false";
+    private static final String updateDriverQuery
+            = "UPDATE drivers SET name = ?, license = ? WHERE id = ?";
+    private static final String deleteDriverQuery
+            = "UPDATE drivers SET is_deleted = true WHERE id = ? AND is_deleted = false";
+
+    private static final String deleteAllDriversQuery
+            = "DELETE FROM drivers";
+    private static final int ID_INDEX = 1;
+    private static final int NAME_INDEX = 2;
+    private static final int LICENSE_INDEX = 3;

```

Comment:
You use the same numbers in different methods for different purpose. I think these constans are redundant.